docker-compose.yml

```diff
@@ -1,13 +1,19 @@
 version: "3"
 
 services:
-  list-app:
-    build: ./list-app
-
   api:
     build: ./api
+    image: prison-wods-api
```

Comment:
This way we can name the resulting image (**prison-wods-api**). If we don't provide this entry the name is autogenerated like this: "projectName_service", in our case, "**prison-wods_api**". Which one do you prefer?